### PR TITLE
SPCTR-2853 Reducing SVG icon file size and localise array in chunks

### PR DESCRIPTION
**Pull request by rahulvarma722**
### Description
- Due to the large size of the Font Awesome icons array, some users are facing an issue where they are unable to edit their WordPress pages. To resolve this issue, we need to optimize the Font Awesome icons array so that it loads more efficiently and does not cause performance issues for users.
- [JIRA](https://brainstormforce.atlassian.net/jira/software/projects/SPCTR/boards/1?assignee=62cf9daea94a6f9c0efeea37&selectedIssue=SPCTR-2853)

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-init-blocks.php [hunk 1]
```diff
@@ -553,6 +553,23 @@ public function register_block_category( $categories, $post ) {
 		);
 	}
 
+	/**
+	 * Localize .
```
Comment (aapatre) [inline classes/class-uagb-init-blocks.php:557]: Please add proper comment for the function

**Review by aapatre — CHANGES_REQUESTED** [review]
1. Please add a proper description like the one mentioned in the JIRA card
2. Also add more code-comments where-ever necessary. (Although there are some comments already, but please go through the file changes and look for sections which might need some context/explanations for future developers)

cc: @rahulvarma722

**Comment by aapatre** [comment]
We can use this description from JIRA @rahulvarma722 with some slight changes:

Due to the large size of the Font Awesome icons array, some users are facing an issue where they are unable to edit their WordPress pages. To resolve this issue, we need to optimize the Font Awesome icons array so that it loads more efficiently and does not cause performance issues for users.

File: Gruntfile.js [hunk 2]
```diff
@@ -1,3 +1,9 @@
+const NUMBER_OF_ICON_CHUNKS = 4;
```
[the remark this answers is not in the document]
Comment (aapatre) [reply Gruntfile.js:5]: Resolved by @rahulvarma722

**Comment by aapatre** [comment]
I've updated the description (the first comment by @rahulvarma722 )

File: classes/class-uagb-helper.php [hunk 3]
```diff
@@ -119,6 +119,22 @@ final class UAGB_Helper {
 		 */
 		public static $table_of_contents_flag = false;
 
+		/**
+		 * As our svg icon is too long array so we will divide that into number of icon chunks.
+		 *
+		 * @var int
+		 * @since X.X.X
```
Comment (AbhijitNage123) [inline classes/class-uagb-helper.php:126]: grunt Bump can not recognize capital X. Can we convert it into small x.x.x? Same for other file

Comment (rahulvarma722) [reply classes/class-uagb-helper.php]: Done

**Review by AbhijitNage123 — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

File: classes/class-uagb-init-blocks.php (hunk 1 above)
Comment (rahulvarma722) [reply classes/class-uagb-init-blocks.php]: Added description

File: blocks-config/uagb-controls/spectra-icons-v6-0.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Font awesome icons array array file.
+ *
+ * @package     Spectra
```
[the remark this answers is not in the document]
Comment (rahulvarma722) [reply blocks-config/uagb-controls/spectra-icons-v6-0.php]: Changed to UAGB

File: blocks-config/uagb-controls/renderIcon.js [hunk 5]
```diff
@@ -17,56 +17,55 @@ function renderSVG( svg, setAttributes = false ) {
...
-			</svg>
-		) : (
-			<svg width="20" height="20" xmlns="https://www.w3.org/2000/svg" viewBox={ viewBox }>
-				<path d={ path }></path>
-			</svg>
-		);
+	switch ( svg ) {
+		case 'align-center':
+			return (
+				<svg viewBox="0 0 18 12" xmlns="https://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (rahulvarma722) [reply blocks-config/uagb-controls/renderIcon.js]: Modified

**Review by adi3890 — APPROVED** [review]

**Pull request merged**